cmd/achcli/describe.go

```diff
@@ -64,13 +64,23 @@ func dumpFiles(paths []string) error {
 	return nil
 }
 
-func readACHFile(path string) (*ach.File, error) {
+func readACHFile(path string, validateOpts *ach.ValidateOpts) (*ach.File, error) {
 	fd, err := os.Open(path)
 	if err != nil {
 		return nil, fmt.Errorf("problem opening %s: %v", path, err)
 	}
 	defer fd.Close()
 
 	f, err := ach.NewReader(fd).Read()
+	if err != nil {
+		return nil, err
+	}
+
+	if validateOpts != nil {
+		if validateErr := f.ValidateWith(validateOpts); validateErr != nil {
```

Comment:
Why not set ValidateOpts on the Reader? 

```suggestion
	r := ach.NewReader(fd)
	r.SetValidation(validateOpts)
	f, err := r.Read()
	if err != nil {
		return nil, err
	}
```